OrbitCore/ElfFileTest.cpp

```diff
@@ -84,7 +92,7 @@ TEST(ElfFile, CalculateLoadBiasNoProgramHeaders) {
   auto elf_file = ElfFile::Create(test_elf_file);
 
   ASSERT_NE(elf_file, nullptr);
-  EXPECT_FALSE(elf_file->GetLoadBias().has_value());
+  EXPECT_FALSE(elf_file->GetLoadBias());
```

Comment:
maybe also check the error message here?

Reply:
done